packages/icons/stories/icon.doc.stories.mdx

```diff
@@ -0,0 +1,103 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Icon, AddDocumentIcon } from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Icons/Icon"
+  component={Icon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Icon
+
+Icon is an important visual representation of an application, a capability, or some other concept or specific entity with meaning on a interface. Icons help users to recognize and understand a concept or an action regardless of linguistic or cultural boundaries.
+
+If an icon is interactive, use [Button](?path=/docs/documentation-core-button--button-variants) (`CTA`, `Primary` or `Secondary` variant).
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon" />
+</Canvas>
+
+## Use Icon when...
+
+- You need a visual representation of a concept.
+- An icon doesn't need to be interactive.
+
+## Features
+
+We have a default icon set that starts from 12px for high- and medium density (MD), then 14px for low density (LD), and 16px for touch density (TD). (Since our base size of high density (HD)/default is 10px but we don’t allow icons size smaller than 12px, hence why here the HD/default will start from 12px, and its 2X is 20px, 3X is 30px etc.)
+You have more size choices by adding multiplies to 2x, 3x, 4x etc.
+
+<Canvas>
+  <Story id="icons-icon--toolkit-icon-multiple-sizes" />
+</Canvas>
+
+We provide two types of icons: default and solid. Some special icons may be excluded from it like `plus` and `loader`.
```

Comment:
At the moment the second sentence here is a little unclear. We should provide a full list of icons that are excluded  - 'Some' is too vague - and explain why they are excluded/special.

Reply:
@pseys Maybe you can provide more context here

Reply:
I checked with @wennazhang22 for context and updated this paragraph

Reply:
Has this question been resolved now? Sorry for the delay, I must've missed a notification. Its not that they're special, they just don't have solid versions because it's not relevant/required. For example, the plus icon doesn't have a 'fillable' aspect so can't have a solid version.

Reply:
@sion-docs Does the content still need to be changed?

Reply:
See the most recent comment below, Fernanda has changed it.